client/mysqlbinlog.cc

```diff
@@ -3489,7 +3489,7 @@ static Exit_status dump_local_log_entries(PRINT_EVENT_INFO *print_event_info,
 
```

Comment:
Status: I fell for the same trap that causes [MDEV-35528](https://jira.mariadb.org/browse/MDEV-35528). Its PR #3670 is planning to move this section to a more orderly place.